drivers/disk/Kconfig.flash

```diff
@@ -4,56 +4,11 @@
 config DISK_DRIVER_FLASH
 	bool "Flash"
 	select FLASH
+	select FLASH_MAP
 	help
 	  Flash device is used for the file system.
 
 if DISK_DRIVER_FLASH
-
-config DISK_FLASH_VOLUME_NAME
```

Comment:
Removal of all these Kconfigs requires entry in release notes.
Information on how to define disk,  within, DTS  should be added to
https://docs.zephyrproject.org/latest/services/storage/disk/access.html.

Reply:
Added both the entry to release notes and a note in disk access documentation.